packages/runtime/src/dataViews/consumption/DecidableRequestItemDVOs.ts

```diff
@@ -20,6 +20,12 @@ export interface DecidableCreateAttributeRequestItemDVO extends DecidableRequest
     attribute: DraftIdentityAttributeDVO | DraftRelationshipAttributeDVO;
 }
 
+export interface DecidableDeleteAttributeRequestItemDVO extends DecidableRequestItemDVO {
+    type: "DecidableDeleteAttributeRequestItemDVO";
+    attributeId: string;
+    attribute: LocalAttributeDVO;
+}
```

Comment:
Just a question for my own understanding: Why don't `Decidable...DVO`s have the `isDecidable` field set to `true`? As it currently stands the `DecidableRequestItemDVO` interface is identical to `RequestItemDVO` and doesn't narrow down any of its fields.

Reply:
From my point of view, it sounds reasonable to have the `isDecidable` property set to `true`. But, I'm not an expert here either (yet 🤓). @jkoenig134 What is your opinion?

Reply:
IMO we can overwrite isDecidable as `isDecidable: true` - yes :)